include/ultra64/viint.h

```diff
@@ -34,4 +37,12 @@
 #define VINTR(v) v
 #define HSTART START
```

Comment:
I'm hesitant to do this since we know what should be in `vi.h` (`os_vi.h`), but we don't have a reference `viint.h` so we can put just about whatever we like in there.

Reply:
Forgetting about the original headers, I think it makes more sense for them to be with the `OSViMode` definition,
but I'll leave it up to you and the "follow the original headers" stance